test-e2e.sh

```diff
@@ -85,10 +85,28 @@ EXCLUDED_TARGETS="-//images/gcloud-bazel:gcloud_install -//images/gcloud-bazel:g
 bazel build -- //... $EXCLUDED_TARGETS
 bazel test  -- //... $EXCLUDED_TARGETS
 
+# Create a unique namespace for this job using the repo name and the build id
+export E2E_NAMESPACE="build-${BUILD_ID:-0}"
+
+kubectl get "namespaces/${E2E_NAMESPACE}" || kubectl create namespace "${E2E_NAMESPACE}"
```

Comment:
Might want to silence errors from the get `kubectl get namespaces/$E &> /dev/null || kubectl create ...` so that these false positive errors don't appear in the logs:

```console
+ kubectl get namespaces/build-332
Error from server (NotFound): namespaces "build-332" not found
```

https://gubernator.k8s.io/build/kubernetes-jenkins/pr-logs/pull/bazelbuild_rules_k8s/208/pull-rules-k8s-e2e/332/